app/src/main/java/com/sun/hero_01/ui/compare/CompareFragment.kt

```diff
@@ -1,11 +1,130 @@
 package com.sun.hero_01.ui.compare
 
+import android.content.Context
+import android.content.SharedPreferences
+import android.content.res.ColorStateList
+import android.graphics.Color
 import android.os.Bundle
 import android.view.View
+import android.widget.ProgressBar
+import android.widget.Toast
 import com.sun.hero_01.R
 import com.sun.hero_01.base.BaseFragment
+import com.sun.hero_01.data.model.HeroDetail
+import com.sun.hero_01.data.source.HeroRepository
+import com.sun.hero_01.data.source.remote.HeroRemoteDataSource
+import com.sun.hero_01.utils.ImageType
+import com.sun.hero_01.utils.extensions.loadHeroImage
+import kotlinx.android.synthetic.main.fragment_compare.*
+import kotlin.math.roundToInt
 
-class CompareFragment {
+class CompareFragment : BaseFragment(), CompareContract.View {
+
+    override val layoutResourceId = R.layout.fragment_compare
+    override var bottomNavigationViewVisibility = View.GONE
+
+    private var comparePresenter: ComparePresenter? = null
+    private val shared = "kotlin shared"
```

Comment:
https://github.com/framgia/coding-standards/blob/master/eng/android/codingstyleguide.md#210-string-constants-naming-and-values